lib/github/middleware.js

```diff
@@ -41,9 +41,13 @@ module.exports = function middleware (callback) {
     enhanceOctokit(context.github, context.log)
 
     newrelic.setControllerName(`github/events.${context.name}`)
+    let webhookEvent = context.name
+    if (context.payload.action !== undefined) {
+      webhookEvent = `${webhookEvent}.context.payload.action`
```

Comment:
```suggestion
      webhookEvent = `${webhookEvent}.${context.payload.action}`
```

I think you wanted this?

On another note, would it make sense to make this `Webhook Name: context.name` and `Action: context.payload.action` (if applicable)?

Reply:
Lol yeah that should be changed.

> On another note, would it make sense to make this Webhook Name: context.name and Action: context.payload.action (if applicable)?

I thought about that. But I preferred the single line dot notation myself. It aligns with how we subscribe to events with the robot.